lib/pipefy_message/providers/aws_client/sns_broker.rb

```diff
@@ -27,18 +28,28 @@ def initialize(_opts = {})
         ##
         # Publishes a message with the given payload to the SNS topic
         # with topic_name.
-        def publish(payload, topic_name)
+        def publish(payload, topic_name, context = "NO_CONTEXT_PROVIDED")
```

Comment:
Is a default value really needed for the `context` argument?

Reply:
I designed it to be an optional parameter and when we don't provide any value `localstack` throws an error for nil values into message attributes (and probably aws sns will have the same behavior).

So, if we keep this parameter as optional, then the answer is yes

Reply:
Ok, thanks!